src/features/common/Layout/UserLayout/UserLayout.tsx

```diff
@@ -338,12 +350,11 @@ function NavLink({
     }
   }
 
-  const [isSubMenuActive, setisSubMenuActive] = React.useState(false);
-
   return (
     <div key={link.title} className={styles.navlinkMenu}>
+      
       <div
-        className={`${styles.navlink} ${activeLink === link.path ? styles.navlinkActive : ''
+        className={`${styles.navlink} ${activeLink === link.path && typeof activeLink !== 'undefined' ? styles.navlinkActive : ''
```

Comment:
Here you can have condition as `activeLink && activeLink === link.path ? <code> :<code>`